src/seedu/addressbook/data/person/PostalCode.java

```diff
@@ -0,0 +1,18 @@
+package seedu.addressbook.data.person;
+
+public class PostalCode {
+    public String postalCodeNumber;
+
+    public PostalCode(String postalCodeNumber) {
```

Comment:
also as mentioned in class, you can delegate validation to each of the component classes